src/HeapDump/HeapDump.csproj

```diff
@@ -0,0 +1,83 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- net451 is actually just another net45 build build with the platform set to x64 -->
```

Comment:
My concern is that this looks like a feature that may never happen ( https://github.com/dotnet/sdk/issues/350 is closed (long ago), and 1938 does not address the issue squarely).    I would not bank on the fix, and even if it comes, it would tie us to a very new version of VS, which is not particularly desirable.   

While small hacks may be reasonable especially if they are time-bounded,, it also seems like the idea of two .csproj files that define one property and include a common .targets file is also very clean, supported now, and about the same total lines of code.   Seems like a win...